doctr/utils/reconstitution.py

```diff
@@ -38,14 +38,18 @@ def synthesize_page(
     # Draw each word
     for block in page["blocks"]:
         for line in block["lines"]:
+            line_ymin = min(int(round(h * word["geometry"][0][1])) for word in line["words"])
+            line_ymax = max(int(round(h * word["geometry"][1][1])) for word in line["words"])
             for word in line["words"]:
                 # Get absolute word geometry
                 (xmin, ymin), (xmax, ymax) = word["geometry"]
                 xmin, xmax = int(round(w * xmin)), int(round(w * xmax))
                 ymin, ymax = int(round(h * ymin)), int(round(h * ymax))
-
                 # White drawing context adapted to font size, 0.75 factor to convert pts --> pix
-                font = get_font(font_family, int(0.75 * (ymax - ymin)))
+                ymin, ymax = line_ymin, line_ymax
+                calculate_font_size = int(0.75 * (ymax - ymin))
```

Comment:
This does still not work well see:

- does still overlap 

![2](https://github.com/mindee/doctr/assets/23098070/29dc07d7-411f-4d84-9677-edafec0b1db4)
![Screenshot from 2024-06-25 09-29-07](https://github.com/mindee/doctr/assets/23098070/0194da09-2784-4bbe-918d-dd9974d2fd9c)
![test_page](https://github.com/mindee/doctr/assets/23098070/8b319e57-c67e-4b7e-9435-1015815ecd93)
![Screenshot from 2024-06-25 09-30-29](https://github.com/mindee/doctr/assets/23098070/24ddc437-1407-418c-bba9-ceaab7993cc8)

Reply:
That is interesting. I shall debug this and see why the issue exists

Reply:
Hey there, what models did you use for these?

Reply:
Hey :)

`fast_base` and `parseq` and `db_mobilenet_v3_large` and `crnn_mobilenet_v3_large`